classes/03class/exercises/c03-docker01/starkmatt/da-terraform

```diff
@@ -0,0 +1,8 @@
+FROM alpine:latest
```

Comment:
Good stuff
Can you just change the name of the file to Dockerfile, which is the default name for a Dockerfile?

Reply:
Roger that!